README.md

```diff
@@ -1,13 +1,27 @@
-# lium - Abstraction Layer of ChromiumOS development
+# lium - Make ChromiumOS development extreamely easy
 
-`lium` is an abstruction layer of ChromiumOS development environment and workflows.
+`lium` is an extreamely user-friendly tool for ChromiumOS developers.
 
-It provides a simple interface for common tasks for ChromiumOS developers,
-with plenty of automatic error recovery mechanisms to avoid keep sticking your eyes on the display(s),
-give you some time for a nap and/or coffee, or other tasks ;)
+It provides a simple way to do common development tasks and make the bareer for contributing to ChromiumOS and lium itself low as much as possible.
+
+It also comes with plenty of automatic error recovery mechanisms to avoid keep sticking your eyes on displays.
 
 Also, it manages local development hardware including DUTs and Servos, and act as working examples of commands to interact with them.
 
+We hope lium gives you some time for a nap and/or coffee, or other tasks by making your work more effective ;)
+
+## Core principles
+
+- P0: Make the basic ChromiumOS development workflow extremely easy as like as a usual software on GitHub
```

Comment:
Make... extremely easy. The goal is to lower the complexity and bar of entry for contribution. It should be as easy to  pull request into lium as any other project on GitHub.